ui/projekt-monsterz-front/src/app/welcome-page/welcome-page.component.css

```diff
@@ -0,0 +1,125 @@
+* {
+     margin: 8px;
+     padding: 8px;
+     z-index: 3;
+}
+button {
+     margin: 0 auto;
+     padding: 20px 35px;
+     border-radius: 20px;
+     border: 2px dotted black;
+}
+div:first-of-type {
+     background-color: #B795C4;
+     color: white;
+}
+div {
+     border-radius: 10px;
+}
+a:hover{
+     cursor: pointer;
+     color: black;
+     
+}
+.welcomeScreenMonster1 {
+     text-align: center;
+     margin: 0 auto;
+     display: flex;
+     flex-direction: column;
+     height: 200px;
+}
+
+.welcomeScreenMonster1Decor {
+     position: relative;
+     max-width: 120px;
+     top: -14px;
+     right: 9px;
+     z-index: 1;
+}
+.welcomeScreenMonster1Eyes {
```

Comment:
So, I take it you personally use camelCase even in CSS/class identifiers? I'm used to kebab-case for it, so I'll have to keep that in mind if I ever touch UI.